### PR TITLE
Port config loading from existing testium

### PR DESCRIPTION
**Pull request by jkrems**
Small change first to make sure all the plumbing is in place and properly working.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -24,5 +26,22 @@
   "bugs": {
     "url": "https://github.com/testiumjs/testium-core/issues"
   },
-  "homepage": "https://github.com/testiumjs/testium-core#readme"
+  "homepage": "https://github.com/testiumjs/testium-core#readme",
+  "devDependencies": {
+    "npub": "^2.2.0",
+    "tap": "^1.4.0"
```
Comment (EndangeredMassa) [inline package.json]: Why tap instead of mocha?

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+language: node_js
+node_js:
+  - 0.10
```
Comment (EndangeredMassa) [inline .travis.yml]: should we make sure this works in 0.12 as well?

File: package.json (hunk 1 above)
Comment (jkrems) [inline package.json]: First and foremost it's "tap tends to be less verbose when using Javascript". Also (quoting myself):

> * tests that do nothing fail
> * files without tests are marked as pending
> * tap reports assertions individually
> * doesn't have to do a fatal failure on every failed assertion
> * also: not a single magical global :heart:

I'm not feeling strongly about it, I'd be fine with using mocha as well. I'm just a bit excited about using tap to implement things like "ok - element visible after 143ms". But for that we only need a tap adaptor, not write the core tests with tap.

File: .travis.yml (hunk 2 above)
Comment (jkrems) [inline .travis.yml]: Good point, added 0.12.

Comment (jkrems) [inline .travis.yml]: Also adding it to `find-open-port`: https://github.com/testiumjs/find-open-port/pull/1

**Comment by jkrems** [comment]
Closing in favor of #2.

**Pull request closed without merging**